proposals/records.md

```diff
@@ -27,6 +27,8 @@ record_body
 Record types are reference types, similar to a class declaration. It is an error for a record to provide
 a `record_base` `argument_list` if the `record_declaration` does not contain a `parameter_list`.
 
+Record parameters cannot use `ref`, `out` or `this` modifiers.
```

Comment:
> (note it doesn't have in yet)

Does that mean we're allowing `in` here, or that we're not allowing `in` here and we'll need to update this when the spec updates? In either case, I think that should be explicitly clarified.

Reply:
`in` is allowed. If it were disallowed, we'd simply list it here as disallowed (no reason not to). I'd rather manage multiple copies of the list of allowed parameter modifiers as it continues to evolve.

Reply:
meant "avoid managing" (ie. the opposite of what I wrote ;-))

Reply:
While I agree in principle, until we have an actual updated master list this just leaves it ambiguous. At some point we'll have to take this and turn it into an actual spec, and at that point we're going to have ask "well, was it intended to be allowed, or was it just not completely spec'ed and we'll do whatever the implementation does?

Reply:
Okay